src/tests/JIT/opt/Compares/compareAnd2Chains.cs

```diff
@@ -178,6 +178,122 @@ public class ComparisonTestAnd2Chains
     public static bool Ge_double_2(double a1, double a2) => a1 >= 5.5 & a2 >= 5.5;
 
 
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    public static void consume<T>(T a1, T a2) {}
+
+    // If conditions that are consumed.
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    public static void Lt_byte_2_consume(byte a1, byte a2) {
+        //ARM64-FULL-LINE: cmp {{w[0-9]+}}, #10
+        //ARM64-FULL-LINE-NEXT: ccmp {{w[0-9]+}}, #11, nc, ge
+        //ARM64-FULL-LINE-NEXT: csel {{w[0-9]+}}, {{w[0-9]+}}, {{w[0-9]+}}, ge
```

Comment:
Can you have the reverse condition too, so that future conditional reversals that semantically are ok do not break this test? I.e. something like `ccmp {{w[0-9]+|#11}}, {{w[0-9]+|#11}}, nc, {{ge|lt}}`, similar in other cases/tests.

Reply:
(I hit this in #81267 too)

Reply:
>something like ccmp {{w[0-9]+|#11}}, {{w[0-9]+|#11}}, nc, {{ge|lt}},

I like this :)